meta/parse.pl

```diff
@@ -2802,6 +2852,55 @@ sub CheckHeadersStyle
             LogWarning "header don't meet style requirements (most likely ident is not 4 or 8 spaces) $header $n:$line";
         }
     }
+
+    if (not -e "/usr/bin/aspell")
+    {
+        LogInfo "ASPELL IS NOT PRESENT, will skip check";
+        return;
+    }
+
+    LogInfo "Running Aspell";
+
+    my @keys = sort keys %wordsToCheck;
+
+    my $count = @keys;
+
+    my $all = "@keys";
+
+    LogInfo "words to check $count";
+
+    my @result = `echo "$all" | /usr/bin/aspell -l en -a`;
+
+    for my $res (@result)
+    {
+        next if not $res =~ /^\s*&\s*(\S+)/;
+
+        my $word = $1;
+
+        chomp $res;
+
+        my $where = "??";
+
+        if (not defined $wordsToCheck{$word})
+        {
+            for my $k (@keys)
+            {
+                if ($k =~/(^$word|$word$)/)
+                {
+                    $where = $wordsToCheck{$k};
+                    last;
+                }
+
+                $where = $wordsToCheck{$k} if ($k =~/$word/);
+            }
+        }
+        else
+        {
+            $where = $wordsToCheck{$word};
+        }
+
+        LogWarning "word '$word' is misspeld $where";
```

Comment:
"Misspelled" is misspelled. ;)

Also probably want to capitalize "W" in "word".